File: swagger/models/get_class_descriptions_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .class_description import ClassDescription

class GetClassDescriptionsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		class_descriptions
		Contains information about the class descriptions.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'ClassDescriptions': ['class_descriptions', ClassDescription],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'class_descriptions': 'ClassDescriptions',
		}

	pagination_response = None
	class_descriptions = None

File: swagger/Models/GetClassDescriptionsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClassDescriptionsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the class descriptions.
	 * 
	 * @var ClassDescription[] 
	 */
	public $classDescriptions;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'ClassDescriptions' => ['classDescriptions', 'array', ClassDescription::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'classDescriptions' => ['ClassDescriptions', 'array', ClassDescription::class],
			];
	}
}
